certified-connectors/LAWLIFT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,660 @@
+            "description": "Success"
+          },
+          "400": {
+            "description": "Bad request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        },
+        "description": "Deletes a LAWLIFT webhook.",
+        "operationId": "DeleteTrigger",
+        "parameters": [
+          {
+            "name": "webhookId",
+            "in": "path",
+            "description": "Webhook Id.",
+            "required": true,
+            "type": "string",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "flowName": {
+                  "type": "string",
+                  "description": "Flow name.",
+                  "title": "Flow Name",
+                  "x-ms-visibility": "important",
+                  "x-ms-summary": "Flow Name"
+                }
+              }
+            }
+          }
+        ],
+        "summary": "Delete Webhook"
+      }
+    }
+  },
+  "definitions": {
+    "Documents": {
+      "type": "object",
+      "description": "Content array of the exported files.",
+      "x-ms-visibility": "advanced",
+      "properties": {
+        "fileContent": {
+          "format": "byte",
+          "type": "string",
+          "title": "File Content",
+          "description": "Content of the exported file.",
+          "x-ms-visibility": "advanced"
+        },
+        "fileName": {
+          "type": "string",
+          "description": "File name of the exported file.",
+          "title": "File Name",
+          "x-ms-visibility": "advanced"
+        },
+        "documentMetadata": {
+          "type": "object",
+          "description": "Document metadata.",
+          "x-ms-visibility": "advanced",
+          "x-schema": {
+            "$ref": "#/definitions/DocumentMetadata"
+          }
+        }
+      }
+    },
+    "TemplateMetadata": {
+      "type": "object",
+      "description": "Publication metadata.",
+      "x-ms-visibility": "advanced",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Template name.",
+          "title": "Template Name",
+          "x-ms-visibility": "advanced"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "title": "Template   created timestamp",
+          "description": "The timestamp that the template was created.",
+          "x-ms  -visibility": "advanced"
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "title": "Template   updated timestamp",
+          "description": "The timestamp that the template was updated.",
+          "x-ms  -visibility": "advanced"
```

Comment:
@makaveli1313 This is a syntax error, Please remove the spaces in tags. This applies anywhere in the connector files.